.github/workflows/e2e-tests.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
-          go-version: ${{ env.GO_VERSION }}
+          go-version: '1.21' 
```

Comment:
interesting 
can you drop the `GO_VERSION` env var definition and check in other places that use that can replace?

thanks!

Reply:
I removed the unused `GO_VERSION` setting in this file / `e2e-tests.yml`.  `go-version` is also set through the `GO_VERSION` environment variable in https://github.com/sigstore/cosign/blob/main/.github/workflows/tests.yaml - but in the corresponding builds, the Go version is set correctly, even though it looks exactly like this one.  Not sure why it doesn't work here...